app/main.py

```diff
@@ -2,0 +1,16 @@
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list[dict], cafe: str) -> None | str:
+    masks_to_buy = 0
+    for friend in friends:
+        try:
+            Cafe(cafe).visit_cafe(friend)
+        except VaccineError:
+            return "All friends should be vaccinated"
+        except NotWearingMaskError:
+            masks_to_buy += 1
+    if masks_to_buy > 0:
+        return f"Friends should buy {masks_to_buy} masks"
+    return f"Friends can go to {cafe.name}"
```

Comment:
The function 'go_to_cafe' lacks a function description. It is a good practice to provide a brief explanation of what the function does, what parameters it takes, and what it returns.